client/me/purchases/remove-plan-dialog/get-plan-features.tsx

```diff
@@ -0,0 +1,257 @@
+import {
```

Comment:
@ciprianimike: Can most of the typos and maintenance concerns be addressed by using and/or extending `[@automattic/calypso-products/plans-list](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx)` and adding something along the lines of `[getCancellationFlowFeatures()](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx#L370)`?

Reply:
> @ciprianimike: Can most of the typos and maintenance concerns be addressed by using and/or extending [@automattic/calypso-products/plans-list](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx) and adding something along the lines of [`getCancellationFlowFeatures()`](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx#L370)?

@michaeldcain I'm afraid it isn't doable within [@automattic/calypso-products/plans-list](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx), I tried adding [getCancellationFlowFeatures()](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx#L370) but there are limits since the list of feature is dynamic and related to the domain. 

So I believe in this case we should go on, implementing @jdc91 suggestion and follow the example of https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

Reply:
The cancellation flow feature list is now integrate in the [@automattic/calypso-products/plans-list](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx) as requested